Gemfile

```diff
@@ -56,6 +56,7 @@ group :development, :test do
   gem 'factory_bot_rails'
   # Add gem for controller Testing.
   gem 'bogus'
+  gem 'devise'
```

Comment:
[OPTIONAL]
- [ ] Please move the devise gem from the `:development, :test` group because Devise is an essential part of the application and should be available in all environments. Having the Devise gem in the `:development` group could potentially lead to issues when deploying the application to production. Therefore, it's crucial to ensure that Devise is accessible in all environments, including production.